### ISSUE TITLE
No sham defined for sn

### ISSUE THREAD

**Issue by fbjork** [nmerouze/machinist_mongo#5]
I'm getting that error when I run a spec.

I'm on Rails 3 beta 4, with machinist_mongo 2.0.0.pre. Any idea what's going on?

**Comment by lennartkoopmann** [comment]
same here!

**Comment by skyeagle** [comment]
You should point bundler to use branch machinist2, e.x.:
    gem 'machinist_mongo', :require => 'machinist/mongoid', :git => 'git://github.com/nmerouze/machinist_mongo.git', :branch => 'machinist2'

It works for me.

**Comment by lennartkoopmann** [comment]
yeah also worked for me! sorry, forgot to update this here. thanks.
